w3-aggregation.md

```diff
@@ -179,7 +184,7 @@ Invoking `aggregate/offer` capability submits an aggregate to a broker service f
 ]
 ```
 
-Each entry of the decoded offers block, has all the necessary information for a Storage Provider to fetch and store a CAR file. The `link` field has the CAR CID, while the `commitmentProof` field has the required `proof` bytes by Storage Providers (for example, `commP`). The `size` field MUST be set to the byte size of the CAR file. The `src` field of each piece MUST be set to a (alphabetically sorted) list of URLs from which it can be fetched. Note that `src` field is optional and can be provided in a different part of the flow such as when deal is signed or through a previously agreed API.
+Each entry of the decoded offers block, has all the necessary information for a Storage Provider to fetch and store a CAR file. The `link` field has the CAR CID, while the `piece` field has the `proof` required by Storage Providers (for example, `commP`). The `src` field of each piece MUST be set to a (alphabetically sorted) list of URLs from which it can be fetched. Note that `src` field is optional and can be provided in a different part of the flow such as when deal is signed or through a previously agreed API.
```

Comment:
> I am less sure about source location (src) at this point, and it starts to feel like a bad idea. Seems to me that putting a routing/redirecting endpoint would buy storefront a lot more flexibility yet not cost too much.

The main motivation to have it there is still the case by what we decided with Mikeal. To decrease need to do any IO operations other than reading the CAR. Anyway, we can discuss that maybe in a separate issue outside of the commp/commd context to avoid noise here?